genesis/src/genesis_accounts.rs

```diff
@@ -569,6 +665,307 @@ pub const VALIDATOR_INFOS: &[ValidatorInfo] = &[
         node_sol: 500.0,
         commission: 0,
     },
+    ValidatorInfo {
+        name: "tangible equipment", // Placeholder name
+        node: "Fe5sLQAAT7RBT8mcH1AAGCbExJQcYxcwXvp1GjrGbvxs",
+        vote: "Dr8MkZZuvZVQJFKtjShZYEfg6n93sc1GxevqLnGss7FW",
+        node_sol: 500.0,
+        commission: 0,
+    },
+    ValidatorInfo {
```

Comment:
Note that I intentionally did not add the `v0.21` placeholder validators to `master`, and as it stands this PR will encounter merge conflicts when lands in master and mergify attempts a backport.  I recommend you rebase this PR on `v0.21` and land it in that branch first -- that's were we need it for Sunday night anyway.  Then manually bring it back to `master` early next week, while removing these placeholder validators